src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlices = createSlice({
+  name: 'catgories',
+  initialState,
+  reducers: {
+    statusCheck: (state, action) => {
+      state.categories = action.payload === 'Under construction' ? 'Under construction' : state.categories;
+    },
+  },
+});
+
+export const { statusCheck } = categoriesSlices.actions;
+
+export default categoriesSlices;
```

Comment:
- [ ] Kindly export default `categoriesSlices.reducer` here instead of just `categoriesSlices`, since you imported the default in the `store` file to use as reducer.